internal/api/create_test_plan_test.go

```diff
@@ -25,6 +25,108 @@ func TestCreateTestPlan(t *testing.T) {
 		t.Error("Error mocking provider", err)
 	}
 
+	params := TestPlanParams{
+		Identifier:  "abc123",
```

Comment:
Just a thought: we don't mock the headers in this request so the Pact contracts don't cover permissions? Or check that a token is sent? This is probably fine, just an observation

Reply:
The interaction expects the token is included in the req header, see line 43.

Reply:
Oh nice lol, bad reading on my part